apps/mobile/src/screens/SettingsScreen/DebugBottomSheet.tsx

```diff
@@ -24,11 +28,21 @@ function DebugBottomSheet(props: Props, ref: ForwardedRef<GalleryBottomSheetModa
     getToken();
   }, []);
 
+  const handleClearStorage = useCallback(() => {
+    AsyncStorage.multiRemove([MARFA_2023_SUBMITTED_FORM_KEY]);
+  }, []);
+
   return (
     <GalleryBottomSheetModal ref={ref} index={0} snapPoints={snapPoints}>
       <View className="flex flex-column space-y-1 mx-4">
         <Typography font={{ family: 'ABCDiatype', weight: 'Bold' }}>Push Token</Typography>
         <Typography font={{ family: 'ABCDiatype', weight: 'Regular' }}>{expoPushToken}</Typography>
+        <Button
```

Comment:
this is for testing - admins can tap this to clear the flag and resubmit the form